app/views/shipments/_form.html.erb

```diff
@@ -18,10 +18,10 @@
         title: 'Taxon name',
         name: 'taxon_name',
         placeholder: "Please select appendix value",
-        data: ['a', 'b', 'c'],
         enabled: true,
         value: (@shipment && @shipment.taxon_name),
-        form: (form && form.object_name)
+        form: (form && form.object_name),
+        apiBaseUrl: Rails.application.secrets.sapi_staging_path
```

Comment:
I don't think I have this setting in secrets.yml? if we need to add it, let's make sure there's no "staging" in the name, we can just call it sapi_path and set separately for each env. Also, if there's anything required in the .env file as well, please add it to .env.example